superset-frontend/src/explore/components/ExploreAdditionalActionsMenu/index.jsx

```diff
@@ -86,8 +86,8 @@ const MenuItemWithCheckboxContainer = styled.div`
 
 const MenuTrigger = styled(Button)`
   ${({ theme }) => css`
```

Comment:
```suggestion
  ${({ theme }) => `
```

Reply:
Is there any reason not to use that? With `css`, my IDE is able to apply css highlighting and suggest css syntax. Without it, it treats the content like regular text

Reply:
It's not a blocker for me, it's just unnecessary because of `styled`. I don't know why the IDE is not highlighting the CSS inside the `styled` tag. If I had to choose I would remove it just because someone reading the code without the same IDE or plugin installed wouldn't understand why there's a `css` clause inside `styled`.